Makefile.am

```diff
@@ -250,8 +251,21 @@ systemdservicedir   = $(systemdsystemunitdir)
 systemdservice_DATA = $(systemdservice_files)
 endif
 
+if AUTOGOPATH
+export GOPATH := $(CURDIR)/.gopath
+PKG_LINK := $(CURDIR)/.gopath/src/github.com/01org/cc-oci-runtime
+
+$(PKG_LINK):
+		rm -rf $(CURDIR)/.gopath
+		mkdir -p  $(CURDIR)/.gopath/src/github.com/01org
```

Comment:
This path is part of `PKG_LINK` and `PKG_`LINK` isn't using `GOPATH`. I'd suggest doing something like:

```
PKG_LINK_BASE := $(GOPATH)/src/github.com/01org
PKG_LINK := $(PKG_LINK_BASE)/cc-oci-runtime

   :
   rm -rf $(GOPATH)
   mkdir -p $(PKG_LINK_BASE)
```